apps/laboratory/tests/canary.spec.ts

```diff
@@ -1,8 +1,15 @@
 import { DEFAULT_SESSION_PARAMS } from './shared/constants'
 import { testMW } from './shared/fixtures/w3m-wallet-fixture'
+import { uploadCanaryResultsToCloudWatch } from './shared/utils/metrics'
+
+const ENV = process.env['ENV'] || 'dev'
+const REGION = process.env['REGION'] || 'eu-central-1'
```

Comment:
where would this be set?

Reply:
by the ECS task https://github.com/WalletConnect/rs-relay/pull/1428